cogs/tags.py

```diff
@@ -60,12 +85,51 @@ async def create(self, ctx, name: lambda inp: inp.lower(), *, text: str):
 
```

Comment:
I think `name: commands.clean_content, *, text: commands.clean_content` then `name = name.lower()` would be clearer than invoking the converter for clean content. (Comment is for line 81 to 84)